accepted/PSR-6-cache-meta.md

```diff
@@ -300,7 +300,7 @@ The following sample code is recommended in order to enforce the type check on
 the expiresAt() method:
 
 ```php
-if (! ($expiration instanceof \DateTime || $expiration instanceof \DateTimeImmutable)) {
+if (! (is_null($expiration) || $expiration instanceof \DateTime || $expiration instanceof \DateTimeInterface)) {
```

Comment:
Why on the former? The two are identical, no?

And is the second a PSR-2 thing?